sdk/python/lib/test/provider/experimental/test_analyzer.py

```diff
@@ -946,74 +1069,6 @@ def __init__(self, args: Args): ...
     )
 
 
-def test_analyze_component_mutually_recursive_complex_types_inline():
```

Comment:
This tests the same as `test_analyze_component_mutually_recursive_complex_types_file`, but since this is inline inside a function, we can't resolve the forward ref for the test comparison.